common/app/controllers/EmailSignupController.scala

```diff
@@ -49,16 +50,15 @@ class EmailFormService(wsClient: WSClient) {
   def submit(form: EmailForm): Future[WSResponse] = {
     val maybeTriggeredSendKey: Option[Int] = listIdsWithTrigger.get(form.listId)
 
-    wsClient.url(Configuration.emailSignup.url).post(
+    val idAccessClientToken = Configuration.id.apiClientToken
+    wsClient.url(Configuration.id.consentEmailerApi)
```

Comment:
Couldn't this just be `Configuration.id.apiRoot + /whatever` and avoid the new config? I am not totally against putting the actual path in config separate to `apiRoot`, but that's not normally something that would be configured in frontend in my experience.

Reply:
Yep that's true, config is unnecessary for this I'd say